tests/cnf/ran/gitopsztp/internal/tsparams/ztpvars.go

```diff
@@ -82,6 +82,9 @@ var (
 		"image-registry-policy-config",
 	}
 
+	// TestLabelKey represents day-2 cluster label key in ClusterInstance CR.
+	TestLabelKey = "custom-test"
```

Comment:
since this is just a string literal it can be put in `consts.go`

Reply:
Okay, updated. Thanks! the change will be in place after submitting a new patch for review.